src/Microsoft.PowerShell.Commands.Diagnostics/AssemblyInfo.cs

```diff
@@ -1,9 +1,6 @@
 using System.Reflection;
 using System.Resources;
 
-[assembly: AssemblyFileVersionAttribute("3.0.0.0")]
-[assembly: AssemblyVersion("3.0.0.0")]
-
 [assembly: AssemblyCulture("")]
 [assembly: NeutralResourcesLanguage("en-US")]
```

Comment:
I don't think `[assembly: AssemblyCulture("")]` is necessary. We can remove it.
If we don't have `<GenerateNeutralResourcesLanguageAttribute>false</GenerateNeutralResourcesLanguageAttribute>` in csproj and remov this attribute from AssemblyInfo.cs, will MSBuild automatically generate the `NeutralResourcesLanguageAttribute(en-US)` attribute in our assemblies?